styles/styles.css

```diff
@@ -212,19 +277,32 @@ main img {
 
 /* sections */
 main .section {
-  padding: 64px 16px;
+  /* section padding/margin for mobile */
+  padding: 0 16px;
+  margin-bottom: 40px;
 }
 
-@media (width >= 600px) {
+/* max width for all wrapper divs */
+.section > div {
+  max-width: var(--max-width);
+  margin: auto;
+}
+
+@media (width >= 900px) {
+  /* max width for all wrapper divs */
+  :root {
+    --max-width: 700px;
```

Comment:
wouldn't `max-width` setting be sufficient for  just `width>=1200`? (can be removed from tablet and mobile ones)
so as we reduce the browser window width, we can handle maybe via padding and margin...